playbooks/common/openshift-master/config.yml

```diff
@@ -154,6 +154,7 @@
   - role: openshift_clock
   - role: openshift_cloud_provider
   - role: openshift_builddefaults
+  - role: openshift_buildoverrides
```

Comment:
probably?  I just copied the openshift_builddefaults role.  Should that also be changed?